odd-platform-api/src/test/resources/scripts/createAll.sql

```diff
@@ -0,0 +1,393 @@
+create table flyway_schema_history
```

Comment:
I suppose we shouldn't have several same migration files, if we will use TRUNCATE instead of dropping table will be enough migrate tables one time when database initialize